lib/util.js

```diff
@@ -14,6 +15,13 @@ import filenamifyUrl from 'filenamify-url';
 import template from 'lodash.template';
 import pify from 'pify';
 import plur from 'plur';
+import type {Options} from './';
+
+// TODO: how can I share this with `index.js`?
+// TODO: is this the right way to add a custom property to a built-in class? I tried to prefix it with `export`, but didn't work.
+declare class PageresStream extends stream$Readable {
+	filename: string;
+}
```

Comment:
I think you should use an [intersection](https://flowtype.org/docs/union-intersection-types.html) type here.

```
type PageresStream = stream$Readable & {filename: string};
```

Not sure though, still very noob on this :p.